backend_addon/{{ cookiecutter.__folder_name }}/pyproject.toml

```diff
@@ -42,129 +125,69 @@ directory = "tests"
 name = "Tests"
 showcontent = true
 
-##
-# Add extra configuration options in .meta.toml:
-#  [pyproject]
-#  towncrier_extra_lines = """
-#  extra_configuration
-#  """
-##
-
-[tool.isort]
-profile = "plone"
-
-##
-# Add extra configuration options in .meta.toml:
-#  [pyproject]
-#  isort_extra_lines = """
-#  extra_configuration
-#  """
-##
-
-[tool.black]
-target-version = ["py38"]
-
-##
-# Add extra configuration options in .meta.toml:
-#  [pyproject]
-#  black_extra_lines = """
-#  extra_configuration
-#  """
-##
-
-[tool.codespell]
-ignore-words-list = "discreet,vew"
-skip = "*.po,*.min.js"
-##
-# Add extra configuration options in .meta.toml:
-#  [pyproject]
-#  codespell_ignores = "foo,bar"
-#  codespell_skip = "*.po,*.map,package-lock.json"
-##
-
-[tool.dependencychecker]
-Zope = [
-  # Zope own provided namespaces
-  'App', 'OFS', 'Products.Five', 'Products.OFSP', 'Products.PageTemplates',
-  'Products.SiteAccess', 'Shared', 'Testing', 'ZPublisher', 'ZTUtils',
-  'Zope2', 'webdav', 'zmi',
-  # ExtensionClass own provided namespaces
-  'ExtensionClass', 'ComputedAttribute', 'MethodObject',
-  # Zope dependencies
-  'AccessControl', 'Acquisition', 'AuthEncoding', 'beautifulsoup4', 'BTrees',
-  'cffi', 'Chameleon', 'DateTime', 'DocumentTemplate',
-  'MultiMapping', 'multipart', 'PasteDeploy', 'Persistence', 'persistent',
-  'pycparser', 'python-gettext', 'pytz', 'RestrictedPython', 'roman',
-  'soupsieve', 'transaction', 'waitress', 'WebOb', 'WebTest', 'WSGIProxy2',
-  'z3c.pt', 'zc.lockfile', 'ZConfig', 'zExceptions', 'ZODB', 'zodbpickle',
-  'zope.annotation', 'zope.browser', 'zope.browsermenu', 'zope.browserpage',
-  'zope.browserresource', 'zope.cachedescriptors', 'zope.component',
-  'zope.configuration', 'zope.container', 'zope.contentprovider',
-  'zope.contenttype', 'zope.datetime', 'zope.deferredimport',
-  'zope.deprecation', 'zope.dottedname', 'zope.event', 'zope.exceptions',
-  'zope.filerepresentation', 'zope.globalrequest', 'zope.hookable',
-  'zope.i18n', 'zope.i18nmessageid', 'zope.interface', 'zope.lifecycleevent',
-  'zope.location', 'zope.pagetemplate', 'zope.processlifetime', 'zope.proxy',
-  'zope.ptresource', 'zope.publisher', 'zope.schema', 'zope.security',
-  'zope.sequencesort', 'zope.site', 'zope.size', 'zope.structuredtext',
-  'zope.tal', 'zope.tales', 'zope.testbrowser', 'zope.testing',
-  'zope.traversing', 'zope.viewlet'
+[tool.ruff]
+target-version = "py311"
```

Comment:
```suggestion
target-version = "py312"
```

Reply:
We should set target-version to the _earliest_ Python version that's supported (3.10), right?